packages/reports/src/Components/indicative-cell.tsx

```diff
@@ -7,20 +8,28 @@ type TIndicativeCell = {
     amount: number;
     contract_info: TContractInfo;
     currency: string;
-    status?: string;
     is_footer: boolean;
     is_sell_requested: boolean;
+    profit: string;
+    status?: string;
 };
 
 const IndicativeCell = observer((props: TIndicativeCell) => {
-    const { amount, contract_info, currency, is_footer, is_sell_requested } = props;
+    const { amount, contract_info, currency, is_footer, is_sell_requested, profit } = props;
     const { portfolio } = useStore();
     const { onClickSell } = portfolio;
 
     return (
         <div className='open-positions__indicative'>
             <div className='open-positions__indicative--amount'>
-                <Money amount={Math.abs(amount)} currency={currency} />
+                <div
+                    className={classNames({
+                        'dc-contract-card--profit': Number(profit) > 0,
+                        'dc-contract-card--loss': Number(profit) < 0,
```

Comment:
just a question: should it be  `Number(profit) <= 0`  for loss maybe? :)

Reply:
I think the idea was not to add color indicator (red or green) when your profit is 0, to make the background neutral

Reply:
Yeah, if profit is 0, it doesn't mean the client is losing anything, so we don't make it red across the app. This is expected behavior.